go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/container-orchestrated-devices/container-device-interface v0.4.0
 	github.com/containernetworking/cni v1.1.1
 	github.com/containernetworking/plugins v1.1.1
-	github.com/containers/buildah v1.26.1-0.20220609225314-e66309ebde8c
+	github.com/containers/buildah v1.26.1
```

Comment:
Per offline discussion, use `go get github.com/containers/buildah@v1.26.1-0.20220609225314-e66309ebde8c` to reverse this change, then use `go mod vendor` and `go mod tidy` to sort out the vendored dependencies.

Reply:
Thanks Nalin, that worked!